problem_builder/public/css/answer.css

```diff
@@ -1,6 +1,6 @@
 .mentoring .answer.editable {
     height: 250px;
-    width: 100%;
+    width: 99%;
```

Comment:
@e-kolpakov Answer blocks look fine in edit mode. The issue that this is supposed to fix manifests itself at unit level and at mentoring level:

![answer-block-broken-display](https://cloud.githubusercontent.com/assets/961441/9824354/7d994c22-58cc-11e5-8165-2b23a8342f1b.png)

The size of the `textarea` exceeds the size of the containing `div`. With the patch applied:

![answer-block-display-fixed](https://cloud.githubusercontent.com/assets/961441/9824379/b3a3fca4-58cc-11e5-8e17-61a165ab6a4a.png)

In both cases, `AnswerBlock.mentoring_view` is called which adds `answer.css` to the fragment. That overrides other rules (e.g., those in `problem-builder-edit.css`). Not sure how else to address this, do you have another idea?